### ISSUE TITLE
OfficeOpenXml Error

### ISSUE THREAD

**Issue by davesexton** [dfinke/ImportExcel#50]
I was trying the following Export-Excel example: 
`Get-Service | Export-Excel "c:\projects\temp\test.xlsx"  -Show -IncludePivotTable -PivotRows status -PivotData @{status='count'}`

And it throws the following error:
`Export-Excel : Unable to find type [OfficeOpenXml.Style.ExcelFillStyle]. Make sure that the assembly that contains this type is loaded.`

**Comment by dfinke** [comment]
Thanks for reporting this Dave. Could you please post a few more details. $PSVersionTable and did you install this on your own, with the iex way from github it from the PowerShell gallery?

Thank you
Doug

**Comment by davesexton** [comment]
Sorry fixed the issue - I am currently working at company that has some
strict policies about what type of files can't be stored in your Documents
folder. As a result when I used the iex method to install I got an error,
so I ran it again but modified it to point at
c:\users\davesexton\powershell\modules. That worked but I forgot to remove
the previous broken install which was missing the DLL file.

All working fine now - works a treat. Keep up the good work.
